implementations/rust/ockam/ockam_core/src/routing/message/transport_message.rs

```diff
@@ -59,8 +67,23 @@ impl TransportMessage {
     pub fn set_tracing_context(self, _tracing_context: OpenTelemetryContext) -> Self {
```

Comment:
Maybe it should be called `link_tracing_context` since we no longer retain the original context

Reply:
That's good idea. I renamed the function to `start_new_tracing_context` to give the idea that we are starting a brand new trace and left a more explicit comment.